python/pacemaker/exitstatus.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2004-2022 the Pacemaker project contributors
+#
+# The version control history for this file may have further details.
+#
+# This source code is licensed under the GNU Lesser General Public License
+# version 2.1 or later (LGPLv2.1+) WITHOUT ANY WARRANTY.
+
+from enum import IntEnum, unique
+
+# These values must be kept in sync with include/crm/common/results.h
+@unique
+class ExitStatus(IntEnum):
```

Comment:
Similarly we could add this class as an "API:" commit (no need to add each enum separately!)

Reply:
Fixed.